extension/lib/doofinder/Client.js

```diff
@@ -32,7 +32,7 @@ class Client {
     })
 
     if (response.statusCode >= 400) {
-      this.log.error(`Doofinder error code ${response.statusCode} in response`, response.body)
+      this.log.error(`Doofinder error code ${response.statusCode} in response`, { body: response.body, request: params, endpoint })
```

Comment:
Flip the error call into 
`this.log.error({ object }, message)` : https://github.com/trentm/node-bunyan#user-content-log-method-api
The fields should go first